lib/ansible/modules/lineinfile.py

```diff
@@ -146,12 +159,24 @@
     regexp: '^SELINUX='
     line: SELINUX=enforcing
 
+- name: Ensure SELinux is set to enforcing mode with literal string
+  lineinfile:
+    path: /etc/selinux/config
+    search_string: 'SELINUX='
+    line: SELINUX=enforcing
+
```

Comment:
I'm not sure this example adds a lot of value.

Reply:
Agree!